version.json

```diff
@@ -1,10 +1,18 @@
 {
-    "MajorVersion": "22.11",
-    "MinorVersion": "004",
+    "MajorVersion": "23.11",
+    "MinorVersion": "001",
     "ReleaseNotes": [
         {
             "author": "omuzzy@ebsco.com",
-            "note": "fixes issues where facets drop upon new selection."
+            "note": "fixes issue where EDS items cannot be sent from cart"
+        },
+        {
+            "author": "omuzzy@ebsco.com",
+            "note": "fixes issue where EDS items cannot be downloaded from cart"
+        },
+        {
+            "author": "omuzzy@ebsco.com",
+            "note": "fixes other minor 23.11 errors"
         }
     ],
     "ShaAdd": "https://widgets.ebscohost.com/prod/api/koha/sha/1711.json"
```

Comment:
Super ignorant question - this file references 1711.json. If you call upon the file it responds with:

```
{
    "edsplugin": {
        "version": [
            {
                "number": "17.11000",
                "sha": "2c43b43ab905685632f14cc38c6d85e174a60833"
            }
        ]
    }
}
```

which also references 1711. It doesn't seem to cause headaches, but I am curious...

Reply:
This has been used for many years as the sha for download verification.  We'll make a note to see if this is still best practice, or if there's a better way, with plans to address in a future PR.